src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task{
+
+    protected String dueDate;
+
+    public Deadline(String description, String dueDate) {
+        super(description);
+        this.dueDate = dueDate;
+    }
+
+    @Override
+    public String toString(){
+        return "[D]"  + super.toString() + " (by:" + dueDate + ")";
+    }
+
```

Comment:
Maybe you could delete this empty line between the two "}", since they are not quite necessary.